wheel/src/api.rs

```diff
@@ -17,6 +17,8 @@ pub const MEMPOOL_MODE: u32 =
 pub fn chia_rs(_py: Python, m: &PyModule) -> PyResult<()> {
     m.add_function(wrap_pyfunction!(a_test_function, m)?)?;
     m.add_function(wrap_pyfunction!(run_generator, m)?)?;
+    m.add_class::<PySpendBundleConditions>()?;
+    m.add_class::<PySpend>()?;
```

Comment:
I believe that you don't actually need to do this. There's not much harm in doing it, but there's also not much benefit since they don't contain any static methods or class methods.

Reply:
surely accessing members end up calling into rust code though, that needs to be exported somehow, no?

Reply:
otherwise the data would have to be copied into python when returned

Reply:
The idea is that you can return `#[pyclass]` objects to python even if you don't export them in the module.